.docker/entrypoint/spec/jekyll_builder_spec.rb

```diff
@@ -81,27 +83,24 @@
 
       before(:all) do
         @thread = Thread.new do
-          jekyll_config_provider = Jekyll::PlantUml::JekyllConfigProvider.new(data_dir)
+          jekyll_env = Jekyll::PlantUml::JekyllEnvironment.new('development', __dir__, data_dir)
+          jekyll_config_provider = Jekyll::PlantUml::JekyllConfigProvider.new(jekyll_env)
           jekyll_config = jekyll_config_provider.provide('serve')
-          jekyll_commander = Jekyll::PlantUml::Commands::JekyllCommander.new(jekyll_config)
-          jekyll_commander.execute('serve')
+          jekyll_server = Jekyll::PlantUml::Commands::JekyllServer.new(jekyll_config)
+          jekyll_server.execute
         end
         @thread.abort_on_exception = true
 
         Jekyll::Commands::Serve.mutex.synchronize do
-          unless Jekyll::Commands::Serve.running?
-            Jekyll::Commands::Serve.run_cond.wait(Jekyll::Commands::Serve.mutex)
-          end
+          Jekyll::Commands::Serve.run_cond.wait(Jekyll::Commands::Serve.mutex) unless Jekyll::Commands::Serve.running?
```

Comment:
Codacy found an issue: [Modifier form of `unless` makes the line too long.](https://app.codacy.com/gh/SwedbankPay/jekyll-plantuml-docker/pullRequest?prid=5786435)